src/plugins/intel_cpu/tests/unit/graph/merge_transpose_reorder_test.cpp

```diff
@@ -151,8 +148,24 @@ class MergeTransposeReorderIsOptimizedCPUTest : public ::testing::Test {
     const ov::Shape testShape{1, 3, 8, 16};
 
     std::unique_ptr<Graph> m_graph;
-}; // class MergeTransposeReorderIsOptimizedCPUTest
+}; // class MergeTransposeReordersCPUTest
+
+// upstream node or downstream node is inPlaced thereby the inserted Reorder cannot be optimized.
+TEST_F(MergeTransposeReordersCPUTest, smoke_Run_MergeTransposeReorders_isOptimized) {
+    CreateGraph(1, Edge::LOOK::LOOK_DOWN);
+    CheckTransposeCount(0);
+    CheckReorderOptimized(std::string("_fake"), false);  // the fused node is of name "reshape_abcd_acdb_fake"
```

Comment:
Hm, this is an interesting outcome. Should we name reorders which are not "optimized" with the "fake" postfix? I think we shouldn't.

Reply:
I don't want to change this convention in my PR since:
 - I'm not sure whether there is other test relying on this naming
 - if we remove _fake, we don't known how to find this reorder (since there are many reorders inserted by other pass/stages)